tests/integration/cattletest/core/test_host_stats.py

```diff
@@ -54,8 +54,8 @@ def test_stats_container(admin_client, docker_context):
     stats_access = container.stats()
 
     assert stats_access.token.index('.') > 0
```

Comment:
I should add that the fallback to uuid in my `getDockerIdentifier()` is for backwards compatibility (to support containers created prior to the initial native-docker commit).